frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -47,7 +48,17 @@ function ReaderCoptListener:onReadSettings(config)
 
     self.old_battery_level = Device:getPowerDevice():getCapacity()
 
-    self:setupHeaderRefresh()
+    -- Have this ready if now or later auto-refresh is enabled
+    self.headerRefresh = function()
+        -- Only draw it if something has changed
+        local new_battery_level = Device:getPowerDevice():getCapacity()
```

Comment:
nit: Comma between "it, if"

Reply:
Sounds fine to me without a comma? (We are talking about the comment, right? ^^).

Reply:
Mhhh, that doesn't really sound how my (so-so) english grammar tells me it should :)
A) I will go out if the weather is good
B) I will go out, if the weather is good.

Calling our english grammar scholars @Frenzie and @NiLuJe :)
May be there's some kind of French (usually no comma) or German (comma) habit that leaked in our englishs ?

Reply:
I mean, if something needs a rewrite, that's the comment *above* this one, because that one is genuinely not great ^^.

e.g., i'd even simplify it to something along the lines of `We need this for auto-refresh support`.

Reply:
"Only draw it if something has changed" is good English.

Reply:
Oh, yay, in there, I would put some commas (as I would in French), but not at the places @zwim might put them :)
```
< Have this ready if now or later auto-refresh is enabled
> Have this ready if, now or later, auto-refresh is enabled
```
But ok with the simpler rewrite.

Reply:
Yeah, that's a fun comma-related example that's indeed slightly better with a bit of Oxford comma thrown in ;p.

But it's even better if you move that to the end, e.g., `Have this ready in case auto-refresh is enabled, now or later`.

(In case you wan to keep a wordier variant).